### DISCUSSION TITLE
Một vài thuật ngữ RL

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#161] [Terminology] [answered]
Em/mình đang dịch chương RL thì gặp những cụm này, thấy gần giống context trong kinh tế. Mn cho ý kiến về cách dịch nhé.
cc: @mlbvn/handson-ml 

`credit assignment problem`: bài toán tính công
`discount factor`: hệ số chiết khấu
`return`: lợi nhuận

Context:
![image](https://user-images.githubusercontent.com/30251584/174899255-52f58076-d3e9-4ff3-b8b1-39c229c95aa4.png)

**Comment by ngcthuong** [comment, marked as answer]
Cái đầu tiên: tính công.

**Comment by ngcthuong** [comment]
Return là lợi nhuận thì phải. Lợi tức là từ khác.

**Reply by cuongvng** [reply]
em tra thấy return có thể dịch là lợi tức hoặc lợi nhuận. 2 từ giống nghĩa nhau 🤔 
https://wiki.tino.org/return-la-gi/

**Reply by ngcthuong** [reply]
https://en.wikipedia.org/wiki/Earnings_yield - cái này mới là lợi tức. Mình không thấy website kia tin cậy lắm.

**Reply by cuongvng** [reply]
Ok vậy dịch là `lợi nhuận` ạ.
